pkg/sql/distsql_plan_csv.go

```diff
@@ -152,6 +152,11 @@ func (c *callbackResultWriter) Err() error {
 
 var colTypeBytes = sqlbase.ColumnType{SemanticType: sqlbase.ColumnType_BYTES}
 
+// KeyRewriter describes helpers that can rewrite keys (possibly in-place).
+type KeyRewriter interface {
+	RewriteKey(key []byte) (res []byte, ok bool, err error)
+}
+
 // LoadCSV performs a distributed transformation of the CSV files at from
 // and stores them in enterprise backup format at to.
 func LoadCSV(
```

Comment:
We should rename this function at some point since it deals with more than CSV now.